src/main/scala/dpla/ingestion3/OaiRecordHarvesterMain.scala

```diff
@@ -121,6 +118,11 @@ object OaiRecordHarvesterMain {
       case (Some(sets), Some(blacklist)) => {
         baseOptions + ("sets" -> (sets.split(",") filter (!blacklist.split(",").contains(_))).mkString(","))
       }
+      // No set list passed in but a blacklist was passed in
+      // TODO: Should this error out or should it try to call ListSets to generate a set list?
```

Comment:
I would be happy to work my refactor into this PR, but the ticket to create a single-invocation OAI harvest isn't in this sprint: https://digitalpubliclibraryofamerica.atlassian.net/browse/DT-1439  We could merge this PR as an "incremental" step, or we could leave it open until I can start work on DT-1439.  Either way is okay with me.